platform/brickpi3-motor-and-sensor.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+"""
+The Brickpi3 doesn't support auto-detecting motors and sensors. To use devices
+connected to the LEGO ports, you must specify what type of device it is.
+"""
+
+from time import sleep
+from ev3dev2 import list_devices
+from ev3dev2.port import LegoPort
+from ev3dev2.motor import OUTPUT_A, LargeMotor, SpeedPercent
```

Comment:
Are the `OUTPUT_A` and `INPUT_1` constants automagically set to the BrickPi3 port names?

Reply:
Yeah! 🎉 

https://github.com/ev3dev/ev3dev-lang-python/blob/ev3dev-stretch/ev3dev2/sensor/__init__.py#L52